src/js/button.js

```diff
@@ -7,13 +12,13 @@
  * @class
  * @constructor
  */
-vjs.Button = vjs.Component.extend({
+var Button = Component.extend({
   /**
    * @constructor
    * @inheritDoc
    */
   init: function(player, options){
```

Comment:
Why would ES6 classes do that?
For one, we are going to be compiled this down to ES5 code, but even then, the new class syntax can be intermixed with what we currently have just the same. The new syntax is just some sugar.